docs/web_advanced.rst

```diff
@@ -896,20 +896,26 @@ background tasks could be registered as an :attr:`Application.on_startup`
 signal handler or :attr:`Application.cleanup_ctx` as shown in the example
 below::
 
-
-  async def listen_to_redis(app):
-      try:
-          sub = await aioredis.create_redis(('localhost', 6379))
-          ch, *_ = await sub.subscribe('news')
-          async for msg in ch.iter(encoding='utf-8'):
-              # Forward message to all connected websockets:
-              for ws in app[websockets]:
-                  ws.send_str('{}: {}'.format(ch.name, msg))
-      except asyncio.CancelledError:
-          pass
-      finally:
-          await sub.unsubscribe(ch.name)
-          await sub.quit()
+  import redis.asyncio as redis
+  from aiohttp import web
+
+
+  async def listen_to_redis(app: web.Application):
+      client = redis.from_url('redis://localhost:6379')
+      pubsub = client.pubsub()
+      channel = 'news'
+      await pubsub.subscribe(channel)
+      while True:
+          try:
+              msg = await pubsub.get_message(ignore_subscribe_messages=True, timeout=1.0)
```

Comment:
Another thing I'm not getting from the documentation. Why do we want a timeout? If we just retry immediately, shouldn't we just wait until actually get a response...?

I might have to open an issue to discuss these examples on the redis-py repo.